docs/markdown/using-snowchain/building-with-snowbridge/running-locally.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+title: Running and Testing the Bridge Locally
+parent: Building With Snowbridge
+permalink: /building-with-snowbridge/running-locally
+nav_order: 1
+---
+
+# Running the bridge locally and testing it end to end manually from UI yourself
```

Comment:
This document is _really_ out of date (if its from the same PR submitted months ago) and will only lead to confusion and frustration for whoever tries to follow it.

I suggest we remove it.